dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1785,6 +1919,43 @@
         "entities"
       ]
     },
+    "EntitiesDocumentResultWithMetadata": {
+      "type": "object",
+      "properties": {
+        "entities": {
+          "type": "array",
+          "description": "Recognized entities in the document.",
+          "items": {
+            "$ref": "#/definitions/EntityWithMetadata"
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DocumentResult"
+        }
+      ],
+      "required": [
+        "entities"
+      ]
+    },
+    "EntityTags": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Tags are to express some similarities/affinity between entities. Entity Tags will be unique globally."
+        },
+        "score": {
```

Comment:
Since you went back to `confidenceScore` to avoid breaking changes, shouldn't you do so here since it's new? It'd be consistent within the same service at least.

Reply:
It looks like this has been addressed, so I will mark it as resolved.